pkg/cfgstruct/bind_test.go

```diff
@@ -150,3 +150,43 @@ func TestBindDevDefaults(t *testing.T) {
 	assertEqual(c.Fields[0].AnotherInt, int(6))
 	assertEqual(c.Fields[3].AnotherInt, int(1))
 }
+
+func TestHiddenDev(t *testing.T) {
+	f := pflag.NewFlagSet("test", pflag.PanicOnError)
+	var c struct {
+		String string `default:"dev" hidden:"true"`
+		Bool   bool   `releaseDefault:"false" devDefault:"true" hidden:"true"`
+		Int64  int64  `releaseDefault:"0" devDefault:"1"`
+		Int    int    `default:"2"`
```

Comment:
Although probably works, I would add a field with `hidden:"false"` for being also test it.

Reply:
good idea. thanks.